asab/library/providers/zookeeper.py

```diff
@@ -31,23 +32,23 @@ def __init__(self, app, path):
 			z_path=self.Path
 		)
 		self.Zookeeper = None
+		self.DisabledPaths =  None
 		self.App.PubSub.subscribe("ZooKeeperContainer.started!", self._on_zk_ready)
+		self.Zookeeper = self.ZookeeperContainer.ZooKeeper
+
+	async def _on_zk_ready(self, event_name, zkcontainer):
+		if zkcontainer == self.ZookeeperContainer:
+			self.DisabledPaths = await self.read(".disabled.yaml")
 
 	async def finalize(self, app):
 		# close client
 		await self.Zookeeper._stop()
 
-	async def _on_zk_ready(self, event_name, zkcontainer):
-		if zkcontainer == self.ZookeeperContainer:
-			self.Zookeeper = self.ZookeeperContainer.ZooKeeper
-
 	async def read(self, path):
 		if self.Zookeeper is None:
```

Comment:
@ateska 

Can I know why we throw this warning ?

This will be caught as an exception correct?

Reply:
The exception text is very misleading.